scripts/end_to_end_test.py

```diff
@@ -140,11 +132,22 @@ def get_default_test_inputs(self, workflow_class: Type[BaseWorkflow]) -> Optiona
             logger.warning(f"Could not generate default inputs for {workflow_class.__name__}: {e}")
             return None
 
-    def push_workflow_to_vellum(self, workflow_class: Type[BaseWorkflow]) -> bool:
-        """Push workflow to Vellum"""
+    def push_workflow_to_vellum(self, module_name: str) -> bool:
+        """Push workflow to Vellum using CLI command"""
         try:
-            logger.info(f"Pushing workflow {workflow_class.__name__} to Vellum")
+            import subprocess
+
+            logger.info(f"Pushing workflow module {module_name} to Vellum")
+
+            result = subprocess.run(
+                ["vellum", "workflows", "push", module_name], capture_output=True, text=True, check=True
+            )
+
+            logger.info(f"Successfully pushed workflow: {result.stdout}")
```

Comment:
remove the result.stdout from logger .info here. We may add it to a data directory, but don't plan on it for now